vars/phaseDeploy.groovy

```diff
@@ -13,18 +13,41 @@ def call(Project project, List<Set<Map>> repos) {
 
     def phase = MROPipelineUtil.PipelinePhases.DEPLOY
 
+    def standardWorkspace = env.WORKSPACE
+    def agentPodCondition = project.isPromotionMode
```

Comment:
It is not clear to me the naming of this boolean variable. Maybe renaming it to `isAgentPod` could help.

Reply:
Hmm, let's leave it like it is for now - the method arg is named the same way.

In any case - this is just a hack. We should work on moving everything to the slave after the release.